lib/rubocop/cop/rspec/factory_bot/excessive_create_list.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Rubocop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Check for create_list FactoryBot declarations
+        # higher than configured MaxAmount.
+        #
+        # @example MaxAmount: 20
+        #   # We do not allow more than 20 items to be created
+        #
+        #   # bad
+        #   create_list(:merge_request, 1000, state: :opened)
+        #
+        #   # good
+        #   create_list(:merge_request, 15, state: :opened)
+        #
+        # @example MaxAmount: 10 (default)
```

Comment:
```suggestion
        # @example
```

Reply:
Please move the default up, so that the MaxAmount conf example section is below it